docs/library/urandom.rst

```diff
@@ -0,0 +1,57 @@
+:mode:`urandom` -- generates random numbers
+===========================================
+
+.. module:: urandom
+   :synopsis: random numbers
+
+This module implements pseudo-random number generator which is a deterministic output.
```

Comment:
"This module implements a pseudo-random number generator."
should be sufficient. If you want to explain, what that means, you have to elaborate it further than with just 5 words. You may insert a link to wiki.